### PR TITLE
chore: Add JetBrains Mono to ThirdPartyLicenses.txt

### PR DESCRIPTION
**Pull request by bryphe**

### CODE REVIEW

**Comment by zbaylin** [comment]
I actually did this in my PR, but I think the format is a little different. I removed Fira Code and put this where it was. Not sure if that was a good idea or not, sorry for not asking!

**Comment by bryphe** [comment]
@zbaylin - perfect! Thanks for updating `ThirdPartyLicenses` 👍

File: ThirdPartyLicenses.txt [hunk 1]
```diff
@@ -3822,3 +3823,209 @@ END OF vscode-codicons NOTICES AND INFORATION
 
 ==============================================
 END OF openssl NOTICES AND INFORATION
```
Comment (rogererens) [inline ThirdPartyLicenses.txt:3825]: Typo: INFORMATION

Comment (bryphe) [reply ThirdPartyLicenses.txt:3825]: Thanks for catching the typo @rogererens - fixing in #1944

**Pull request closed without merging**